shared/js/angularProjects.js

```diff
@@ -1,82 +1,83 @@
-  'use strict';
+/* global $, angular */
```

Comment:
could these be provided by `require` statements now that this is bundled through browserify, or is it still being used by non-commonjs code? e.g.:
```
var $ = require('jquery');
var angular = require('angular');
```